content/blog/2022-05-31-may-22-community-gems.md

```diff
@@ -0,0 +1,188 @@
+---
+title: May '22 Community Gems
+date: 2022-05-31
+description: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with CML and GCP, DVC data and remotes, DVC pipelines and setups, and
+  more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with CML and GCP, DVC data and remotes, DVC pipelines and setups, and
+  more.
+picture: 2022-05-31/may-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/may-22-community-gems/1184
+tags:
+  - DVC Remotes
+  - Pipelines
+  - CML
+  - GCP
+  - Community
+---
+
+### [Is it possible to export a plot generated using `dvc plots diff HEAD main` with multiple files to vega-lite?](https://discord.com/channels/485586884165107732/563406153334128681/965911829538832435)
+
+Thanks for the awesome question @dominic!
+
+If you are working with multiple files like `prediction.json` and
+`main.prediction.json`, you can still use the `dvc plots diff` command to get
+both of the plots to show on a single graph. You'll need to run the following
+command:
+
+```dvc
+$ dvc plots diff HEAD main --targets prediction.json main.prediction.json --json
+```
+
+We are working on expanding `dvc plots` to allow you to compare data from two
+files. You can also include this plot in a comment with CML so that it appears
+on your pull requests in GitHub.
```

Comment:
```suggestion
You can also include this plot in a comment with CML so that it appears
on your pull requests in GitHub.
```

Maybe also include a CML example or link to https://cml.dev/doc/cml-with-dvc.